webapp/components/interop-summary.js

```diff
@@ -221,7 +221,13 @@ class InteropSummary extends PolymerElement {
     this.updateSummaryScore(scoreNumbers[0], scores[scores.length - 1][summaryFeatureName]);
     // Update the rest of the browser scores.
     for (let i = 1; i < scoreNumbers.length; i++) {
-      this.updateSummaryScore(scoreNumbers[i], scores[i - 1][summaryFeatureName]);
+      let browserScore = scores[i - 1][summaryFeatureName];
+      // For Interop 2022, 10% of the browser score was based on the investigation scores.
+      if (this.year === '2022') {
+        const investigationScore = this.dataManager.getYearProp('investigationTotalScore');
```

Comment:
It looks like this mixes in the investigation score equally throughout the year and not based on when the investigation score changes as the dashboard originally did. Is that right?

Reply:
Good catch! That is correct that this change only adjusts the scores at the end and will not reflect incremental changes in the graph. I will revert to the old implementation for 2022 in order to keep the graph consistent.